Uncompressing Canada 2011 census fails

It looks like the file has changed with an incompatible format:

```
✗ make -- run ca.statcan.data.DownloadData --resolution da_ --survey census
docker-compose run --rm bigmetadata luigi --module tasks.ca.statcan.data tasks.ca.statcan.data.DownloadData --resolution da_ --survey census
Starting bigmetadata_postgres_1 ... done
Starting bigmetadata_bigmetadata_daemon_1 ... done
INFO: Starting new HTTP connection (1): bigmetadata_daemon
INFO: Informed scheduler that task   tasks.ca.statcan.data.DownloadData_da__census_c30a093026   has status   PENDING
INFO: Done scheduling tasks
INFO: Running Worker with 1 processes
INFO: [pid 1] Worker Worker(salt=846184023, workers=1, host=606fcbf120ed, username=root, pid=1) running   tasks.ca.statcan.data.DownloadData(resolution=da_, survey=census)
2018-02-19 13:37:48,706 Downloading http://www12.statcan.gc.ca/census-recensement/2011/dp-pd/prof/details/download-telecharger/comprehensive/comp_download.cfm?CTLG=98-316-XWE2011001&FMT=CSV1501
INFO: Downloading http://www12.statcan.gc.ca/census-recensement/2011/dp-pd/prof/details/download-telecharger/comprehensive/comp_download.cfm?CTLG=98-316-XWE2011001&FMT=CSV1501
ERROR: [pid 1] Worker Worker(salt=846184023, workers=1, host=606fcbf120ed, username=root, pid=1) failed    tasks.ca.statcan.data.DownloadData(resolution=da_, survey=census)
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/luigi/worker.py", line 194, in run
    new_deps = self._run_get_new_deps()
  File "/usr/local/lib/python3.5/dist-packages/luigi/worker.py", line 131, in _run_get_new_deps
    task_gen = self.task.run()
  File "/bigmetadata/tasks/base_tasks.py", line 423, in run
    self.uncompress()
  File "/bigmetadata/tasks/base_tasks.py", line 452, in uncompress
    z.extractall(path='{output}'.format(output=self.output().path))
  File "/usr/lib/python3.5/zipfile.py", line 1347, in extractall
    self.extract(zipinfo, path, pwd)
  File "/usr/lib/python3.5/zipfile.py", line 1335, in extract
    return self._extract_member(member, path, pwd)
  File "/usr/lib/python3.5/zipfile.py", line 1397, in _extract_member
    with self.open(member, pwd=pwd) as source, \
  File "/usr/lib/python3.5/zipfile.py", line 1318, in open
    return ZipExtFile(zef_file, mode, zinfo, zd, True)
  File "/usr/lib/python3.5/zipfile.py", line 718, in __init__
    self._decompressor = _get_decompressor(self._compress_type)
  File "/usr/lib/python3.5/zipfile.py", line 648, in _get_decompressor
    raise NotImplementedError("compression type %d (%s)" % (compress_type, descr))
NotImplementedError: compression type 9 (deflate64)
INFO: Informed scheduler that task   tasks.ca.statcan.data.DownloadData_da__census_c30a093026   has status   FAILED
INFO: Worker Worker(salt=846184023, workers=1, host=606fcbf120ed, username=root, pid=1) was stopped. Shutting down Keep-Alive thread
INFO:
===== Luigi Execution Summary =====

Scheduled 1 tasks of which:
* 1 failed:
    - 1 tasks.ca.statcan.data.DownloadData(resolution=da_, survey=census)

This progress looks :( because there were failed tasks

===== Luigi Execution Summary =====

make: *** [run] Error 1
```

Download [the file](http://www12.statcan.gc.ca/census-recensement/2011/dp-pd/prof/details/download-telecharger/comprehensive/comp_download.cfm?CTLG=98-316-XWE2011001&FMT=CSV1501) and run this in `python3` CLI to reproduce:

```python
import zipfile
path = '98-316-XWE2011001-1501_CSV.zip'
with zipfile.ZipFile(path) as z:
    z.extractall(path='.')
```

--- a/tasks/base_tasks.py
+++ b/tasks/base_tasks.py
@@ -448,8 +448,20 @@ class DownloadUnzipTask(DownloadUncompressTask):
     '''
 
     def uncompress(self):
-        with zipfile.ZipFile('{output}.zip'.format(output=self.output().path), 'r') as z:
-            z.extractall(path='{output}'.format(output=self.output().path))
+        output = self.output().path
+        LOGGER.debug("Uncompressing {output}".format(output=output))
+        try:
+            with zipfile.ZipFile('{output}.zip'.format(output=output), 'r') as z:
+                z.extractall(path='{output}'.format(output=output))
+        except NotImplementedError as err:
+            s_err = str(err)
+            LOGGER.warn("%s.zip error: %s. Fallback to command line...", output, s_err)
+            if s_err == 'compression type 9 (deflate64)':
+                # Support for unsupported file types, such as PKWare deflate64 format
+                subprocess.check_call(['unzip', '{output}.zip'.format(output=output)])
+            else:
+                raise
+
 
 
 class DownloadGUnzipTask(DownloadUncompressTask):

